codec/src/main/java/io/netty/handler/codec/compression/FastLz.java

```diff
@@ -168,7 +168,7 @@ static int compress(final byte[] input, final int inOffset, final int inLength,
                         readU16(input, inOffset + ip - 1) == readU16(input, inOffset + ip + 1)) {
                     distance = 1;
                     ip += 3;
-                    ref = anchor - 1 + 3;
+                    ref = anchor + (3 - 1);
```

Comment:
This might also be an opportunity to just do `anchor + 2`, but explain in a comment where the `2` comes from.

Reply:
I'm not sure how to formulate the correct comment here..

Reply:
I'll confess that I'm not sure, either.